src/Core/Domain/Order/QueryResult/OrderProductForViewing.php

```diff
@@ -306,4 +320,20 @@ public function isRefundable(): bool
 
         return true;
     }
+
+    /**
+     * @return int
+     */
+    public function getOrderInvoiceId(): ?int
+    {
+        return $this->orderInvoiceId;
+    }
+
+    /**
+     * @return string
```

Comment:
Can you please add some comments on ALL those getters to clarify this class

Reply:
done